qiskit_cold_atom/circuit_to_cold_atom.py

```diff
@@ -19,159 +19,179 @@
 from qiskit_cold_atom.exceptions import QiskitColdAtomError
 
 
-def validate_circuits(
-    circuits: Union[List[QuantumCircuit], QuantumCircuit],
-    backend: Backend,
-    shots: Optional[int] = None,
-) -> None:
-    """
-    Performs validity checks on circuits against the configuration of the backends. This checks whether
-    all applied instructions in the circuit are accepted by the backend and whether the applied gates
-    comply with their respective coupling maps.
-
-    Args:
-        circuits: The circuits that need to be run.
-        backend: The backend on which the circuit should be run.
-        shots: The number of shots for each circuit.
-
-    Raises:
-        QiskitColdAtomError: - If the maximum number of experiments or shots specified by the backend is
-                             exceeded
-                             - If the backend does not support an instruction given in the circuit
-                             - If the width of the circuit is too large
-                             - If the circuit has unbound parameters
+class CircuitTools:
+    """A class to provide tooling for cold-atomic circuits.
```

Comment:
Maybe this file should be renamed to `circuit_tools.py` now that we have this nice wrapper class?

Reply:
I like that. See here: d3b6003